src/glide/browser/base/content/browser-hints.mts

```diff
@@ -75,6 +75,11 @@ class GlideHintsClass {
 
     const hints = picked_hints.map((hint): GlideResolvedHint => ({ ...hint, label: "" }));
 
+    // clear "No hints found" notification when hints are found
```

Comment:
nit: this comment isn't really useful because it just says the same thing as the code, it'd be better for it to say *why* we should clear the "No hints found" notification instead